### DISCUSSION TITLE
传递formdata格式的数据浏览器显示出错

### DISCUSSION THREAD

**Discussion by day-xue** [alovajs/alova#65] [Q&A]
<img width="658" alt="image" src="https://user-images.githubusercontent.com/95938844/236615828-6898b05f-c1c2-466a-a569-71cf20880d96.png">

<img width="683" alt="image" src="https://user-images.githubusercontent.com/95938844/236615860-83babc39-0554-4b0b-952c-422f479b74c1.png">

<img width="487" alt="image" src="https://user-images.githubusercontent.com/95938844/236615886-9b5e302f-ed5b-425b-bf4d-ec09153fb884.png">

**Comment by day-xue** [comment]
<img width="748" alt="image" src="https://user-images.githubusercontent.com/95938844/236615938-82f9056f-6037-4c23-a720-88f7e5ed7e6f.png">

axios下面的formdata 浏览器显示是这样的

**Comment by day-xue** [comment]
换成application/x-www-form-urlencoded 也不行

**Comment by JOU-amjs** [comment]
@day-xue 你使用的是`@alova/adapter-xhr`吗？如果是的话，目前已经修复了，请更新到v1.0,1

**Reply by wu-yiqiang** [reply]
我用的是alova/fetch，也存在类似问题，请问该如何解决？

**Reply by MeetinaXD** [reply]
data对象得是一个FormData对象才可以
